gui/guihtml/src/TGHtmlParse.cxx

```diff
@@ -1103,8 +1102,7 @@ void TGHtml::TokenizerAppend(const char *text)
       return;
    }
 
-   // coverity[secure_coding]
-   strcpy(&fZText[fNText], text);
+   strlcpy(&fZText[fNText], text, strlen(text+1));
```

Comment:
Also here one should have something like `sizeof(fZText) - fNText` instead of `strlen(text+1)`